schema/filter_artworks.js

```diff
@@ -80,6 +80,9 @@ const FilterArtworks = {
     major_periods: {
       type: new GraphQLList(GraphQLString),
     },
+    partner_id: {
+      type: GraphQLString,
```

Comment:
per @alloy , this should maybe be `GraphQLID`? (it can def be null tho so doesn't need the `GraphQLNonNull` part).